### PR TITLE
Movies creation

### PR DESCRIPTION
**Pull request by ffasitella**
### Summary

- Added MoviesController
- Added get and post methods
- Creation of new movies in the db

### Trello card

[https://trello.com/c/issYdHzB/8-creado](https://trello.com/c/issYdHzB/8-creado)

### CODE REVIEW

File: src/Resources/en-US.json [hunk 1]
```diff
@@ -3,5 +3,6 @@
     "ContactPage": "Your contact page.",
     "HomePage": "Home Page",
     "PrivacyPolicy": "Privacy Policy",
-    "MovieList": "Movie List"
+    "MovieList": "Movie List",
+    "CreateTitle": "Create a new movie"
 }
```
Comment (gzamudio) [inline src/Resources/en-US.json]: add empty line

File: src/Controllers/MoviesController.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+{
+    [Route("[controller]")]
+    public class MoviesController : Controller
+    {
+        private readonly IUnitOfWork _unitOfWork;
+        public IUnitOfWork UnitOfWork { get { return this._unitOfWork; } }
+
+        public MoviesController(IUnitOfWork unitOfWork)
+        {
+            this._unitOfWork = unitOfWork;
```
Comment (gzamudio) [inline src/Controllers/MoviesController.cs]: remember to use body expressions

File: src/Views/Movies/Create.cshtml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+@using Microsoft.AspNetCore.Mvc.Localization
+@using System.Globalization
+@inject IViewLocalizer Localizer
+@model MvcMovie.Models.Views.MovieViewModel
+
+@{
+    ViewData["Title"] = Localizer["CreateTitle"];
+}
+
+<form asp-action="Create">
```
Comment (gzamudio) [inline src/Views/Movies/Create.cshtml:10]: its ok for now, but next time try to use [html helpers](http://www.tutorialsteacher.com/mvc/html-helpers)

**Review by gzamudio — APPROVED** [review]

File: src/appsettings.Development.json [hunk 4]
```diff
@@ -7,5 +7,5 @@
       "Microsoft": "Information"
```
Comment (mar7in91Wolox) [inline src/appsettings.Development.json]: This file should be in the gitignore :)

File: src/Models/Views/MovieViewModel.cs [hunk 5]
```diff
@@ -0,0 +1,16 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace MvcMovie.Models.Views
+{
+    public class MovieViewModel
```
Comment (mar7in91Wolox) [inline src/Models/Views/MovieViewModel.cs:6]: Be careful. I think there are a few attributes that should be `Required`, for example, the Title (maybe the others, also). It depends on the business model. Use Data Annotations :)

**Review by mar7in91Wolox — CHANGES_REQUESTED** [review]

**Pull request merged**